layout/_third-party/comments/valine.swig

```diff
@@ -1,15 +1,15 @@
 {%- set valine_uri = theme.vendors.valine or '//unpkg.com/valine/dist/Valine.min.js' %}
 
 <script>
-NexT.utils.loadComments(() => {
+NexT.utils.loadComments(document.querySelector('#valine-comments'), () => {
   NexT.utils.getScript('{{ valine_uri }}', () => {
     var GUEST = ['nick', 'mail', 'link'];
     var guest = '{{ theme.valine.guest_info }}';
     guest = guest.split(',').filter(item => {
       return GUEST.includes(item);
     });
     new Valine({
-      el: '#comments',
+      el: '#valine-comments',
```

Comment:
this makes the hyperlink doesn't working.
https://github.com/theme-next/hexo-theme-next/blob/34ffcee4a73155133b1b0049dcaf816a00703d00/scripts/filters/comment/valine.js#L28